src/modules/help.ts

```diff
@@ -0,0 +1,85 @@
+import { command, default as CookiecordClient, Module } from 'cookiecord';
+import { Message, MessageEmbed } from 'discord.js';
+
+export default class HelpModule extends Module {
+	protected client: any;
```

Comment:
This definition is implicit (it's in `Module`) and even if it wasn't, you can use the `CookiecordClient` type for this instead.